prime-router/src/main/kotlin/common/HttpClientUtils.kt

```diff
@@ -312,35 +330,38 @@ class HttpClientUtils {
             httpClient: HttpClient? = null,
         ): HttpResponse {
             return runBlocking {
-                (httpClient ?: createDefaultHttpClient(accessToken)).submitForm(
-                    url,
-                    formParameters = Parameters.build {
-                        formParams?.forEach { param ->
-                            append(param.key, param.value)
+                // use{...} will automatically close the client
```

Comment:
Let's remove these comments because use is not anything special to this library, it works on anything that is `Closable`. see: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html. We should prefer not to document Kotlin features, it's easy enough to look up imo.

Reply:
ack